docs/01-app/03-api-reference/04-functions/expirePath.mdx

```diff
@@ -0,0 +1,123 @@
+---
+title: expirePath
+description: API Reference for the expirePath function.
+---
+
+`expirePath` allows you to purge [cached data](/docs/app/building-your-application/caching) on-demand for a specific path.
+
+> **Good to know**:
+>
+> - `expirePath` is available in both [Node.js and Edge runtimes](/docs/app/building-your-application/rendering/edge-and-nodejs-runtimes).
+> - `expirePath` only invalidates the cache when the included path is next visited. This means calling `expirePath` with a dynamic route segment will not immediately trigger many expirations at once. The invalidation only happens when the path is next visited.
+> - Currently, `expirePath` invalidates all the routes in the [client-side Router Cache](/docs/app/building-your-application/caching#client-side-router-cache) when used in a server action. This behavior is temporary and will be updated in the future to apply only to the specific path.
```

Comment:
Q: Do we have a warning for that? Sounds like easily being misused if the path is set, but all client cache is invalidated?

Reply:
That would be a warning for any time `expirePath()` or `revalidatePath()` is used in a server action so too noisy.